df/plugin.go

```diff
@@ -84,6 +84,12 @@ var (
 		"securityfs",
 		"devpts",
 		"mqueue",
+		"hugetlbfs",
+		"nsfs",
```

Comment:
Attached is the output of /host-proc/1/mountinfo where /host-proc is the /proc from the host as seen within a K8 container
[k8s_mountinfo.txt](https://github.com/intelsdi-x/snap-plugin-collector-df/files/462207/k8s_mountinfo.txt)
This explains why we want to exclude that many "non-relevant" host filesystems monitored from within the container
Hope this makes sense